google/datalab/ml/_util.py

```diff
@@ -13,10 +13,9 @@
 # limitations under the License.
 
 
-from apache_beam.io import gcsio
-import glob
 from googleapiclient import discovery
-import logging
+from tensorflow.python.lib.io import file_io as tfio
```

Comment:
Can we use tf.gfile, which seems come with public document (https://www.tensorflow.org/api_docs/python/tf/gfile/GFile).